### ISSUE TITLE
Add ability to have 2 status lines

### ISSUE THREAD

**Issue by huggybearbass** [catppuccin/tmux#220]
### Is there an existing issue outlining your improvement?

- [X] I have searched the existing issues and my improvement has not been raised yet.

### What would you like to see added and/or changed?

I love the theme, but I would relish an easy way to set-option -g status 2 and place the themed content into the  second status bar. For anyone who is wondering what I'm talking about, temporarily disable the cappuccin theme and set this, then reload tmux:
set-option -g status 2
set -g status-format[0] '#[align=left]Top left'
set -ag status-format[0] '#[align=centre]Top center'
set -ag status-format[0] '#[align=right]Top right'
set -g status-format[1] '#[align=left]Bottom left'
set -ag status-format[1] '#[align=centre]Bottom center'
set -ag status-format[1] '#[align=right]Bottom right'

If I could utilize those positions it would be incredible!

**Comment by vdbe** [comment]
See #192

**Comment by huggybearbass** [comment]
Thanks! I was able to get the line to show up. How would I go about putting some of my items in the lower line?

**Comment by vdbe** [comment]
I am guessing `set -g status-format[0 or 1] 'whatever you want'`

catppuccin itself will only use 1 line

**Comment by huggybearbass** [comment]
Ahh, ok. Well, that's why I'm asking for a feature, to have catppuccin use 2 lines

**Comment by johnend** [comment]
Yeah, was just looking for a way to have Catppuccin on one line and another below it. It would be great to be able to format the `status-format[0]` for example with Catppuccin and then have `status-format[1]` be empty to create a break between the status line and the top of NeoVim for example.

**Comment by kjnsn** [comment]
The theme colors are available for formatting, so you can do things like `set -g status-format[1] '#[bg=#{@thm_mantle},fg=#{@thm_fg}] <your stuff here>'`.

Is there something specific you'd like to add to those status lines? We would love to see what you create on https://github.com/catppuccin/tmux/discussions/317 :)
